mountpoint-s3-client/tests/copy_object.rs

```diff
@@ -0,0 +1,77 @@
+#![cfg(feature = "s3_tests")]
+
+pub mod common;
+use aws_sdk_s3::primitives::ByteStream;
+use bytes::Bytes;
+use common::creds::get_sdk_default_chain_creds;
+use common::*;
+use mountpoint_s3_client::config::{EndpointConfig, S3ClientAuthConfig, S3ClientConfig};
+use mountpoint_s3_client::error::ObjectClientError;
+use mountpoint_s3_client::S3RequestError;
+use mountpoint_s3_client::{ObjectClient, S3CrtClient};
+use mountpoint_s3_crt::auth::credentials::{CredentialsProvider, CredentialsProviderStaticOptions};
+use mountpoint_s3_crt::common::allocator::Allocator;
+#[tokio::test]
+async fn test_copy_objects() {
+    let sdk_client = get_test_sdk_client().await;
+    let (bucket, prefix) = get_test_bucket_and_prefix("test_copy_object_prefix1");
```

Comment:
You should use the test name:
```suggestion
    let (bucket, prefix) = get_test_bucket_and_prefix("test_copy_objects");
```

Reply:
ack